packages/unified-pipeline/src/components/CircleOverlay/CircleOverlay.tsx

```diff
@@ -26,7 +26,7 @@ const CircleOverlay = (props: CircleOverlayInterface) => {
       {circles.map((circle, index) => (
         <div
           key={index}
-          className={`w-[${circle.size}] h-[${circle.size}] absolute translate-x-[-50%] translate-y-[-50%] rounded-full bg-[rgba(255,255,255,${circle.opacity})]`}
+          className={`w-[${circle.size}] h-[${circle.size}] absolute translate-x-[-50%] translate-y-[-50%] rounded-full bg-white/[${circle.opacity}]`}
```

Comment:
let's not use `white` as it wont work with inverted theme

Reply:
Will fix this, currently this component is not rendered anywhere, just part of code base.